fluid-project/trivet
# fix: resolve issues with localized pagination

## Description

I noticed some of the strings in the pagination component weren't localized. Then, in testing them, I realized that Eleventy's pagination plugin doesn't appear to wait for the computed data and as such sets permalinks incorrectly in the pagination links (the pages were built in the right place so I didn't realize this was happening). So for the posts archive pages, we need to set a permalink in the front matter. This shouldn't change any instructions.

## Steps to test

1. Add some more posts in French and English.
2. Change the pagination size in `src/collections/pages/**/posts.md` to `1`.

**Expected behaviour:** Pagination works as expected in both English and French.

## Additional information

Not applicable.

## Related issues

Not applicable.
